### DISCUSSION TITLE
Customize Airflow UI

### DISCUSSION THREAD

**Discussion by dharm187** [apache/airflow#18078] [Q&A]
I was looking for some documentation to customize airflow ui. for ex. change the airflow icon to another one, but can't find any thing.

Is there a feature which allows this customization?

**Comment by turbaszek** [comment]
As far as I can tell there's only option to change navbar colour:
https://github.com/apache/airflow/blob/b658a4243fb5b22b81677ac0f30c767dfc3a1b4b/airflow/config_templates/default_airflow.cfg#L590-L591

If you need more customisation you can always patch Airflow css and webserver to achieve desired results.

**Comment by gpadavala** [comment]
how to do css patching? @turbaszek 

I am extending airflow image using Dockerfile?
